WORKSPACE

```diff
@@ -27,14 +24,27 @@ workspace(
 load("//dependencies/vaticle:repositories.bzl", "vaticle_dependencies")
 vaticle_dependencies()
 
-# Load //builder/bazel for RBE
-load("@vaticle_dependencies//builder/bazel:deps.bzl", "bazel_toolchain")
-bazel_toolchain()
+# Load //builder/python
+load("@vaticle_dependencies//builder/python:deps.bzl", python_deps = "deps")
+python_deps()
+
+load("@rules_python//python:repositories.bzl", "py_repositories", "python_register_toolchains")
+py_repositories()
+python_register_toolchains(
+    name = "python39",
+    python_version = "3.9",
+)
```

Comment:
newer version of `rules_python` allows specifying a 'hermitic' python toolchain, which may or may not be respected by rules requiring python.

In fact, our python grpc rules do not, so we apply a patch to them in `dependencies` to facilitate providing a python toolchain